Source/WebCore/page/scrolling/ScrollingCoordinator.h

```diff
@@ -33,6 +33,7 @@
 #include "ScrollSnapOffsetsInfo.h"
 #include "ScrollTypes.h"
 #include "ScrollingCoordinatorTypes.h"
+#include "UserInterfaceLayoutDirection.h"
```

Comment:
You should be able to forward-declare the enum.

Reply:
I couldn't find a good way to do this since it's needed on Page.